azure-pipelines.yml

```diff
@@ -170,30 +170,35 @@ jobs:
 
 - job: Release
   dependsOn: Build
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
   pool:
     vmImage: ubuntu-16.04
   steps:
-  - task: UsePythonVersion@0
-    displayName: Select Python 3.6
-    inputs:
-      versionSpec: '3.6'
   - task: Bash@3
-    displayName: Check git tag
+    displayName: git tag == git branch
     inputs:
       targetType: inline
       script: |
-        set -x
-        TAG=$(git describe --tags)
-        echo "##vso[task.setvariable variable=TAG]$TAG"
```

Comment:
in hindsight, maybe it was the trailing double quote here, I dunno. it's no longer an issue anymore :)